app/models/shop.rb

```diff
@@ -55,6 +55,14 @@ def self.collection
     DB[:shops]
   end
 
+  def to_json(options={})
```

Comment:
The parameter `options={}` has not been used within the method. So it is probably redundant for now

Reply:
Also, i don't see any invocations of this to_json method 🤔

Reply:
@actfong apparently, sinatra uses it under the hood. if I remove it, I got shop instances on `/v1/shops` instead of the json output.

Reply:
OK. So can the argument be removed? Or it can't be removed either?

Reply:
I tried to get rid of it but I got an error:

```
ArgumentError at /v1/shops
wrong number of arguments (given 1, expected 0)
file: shop.rb location: to_json line: 58
```

I didn't find any docs about it though. for now we can keep it as:

```ruby
def to_json(_options={})
```

@mlan080 where did you find instructions for this method usage?